### DISCUSSION TITLE
No scrollbars are shown on sidebars

### DISCUSSION THREAD

**Discussion by barreeeiroo** [squidfunk/mkdocs-material#2797] [Q&A] [answered]
We have recently updated to the latest version of mkdocs-material our documentation page. But, suddenly, both sidebars no longer have scrollbars and became fixed without being able to read the bottom elements nor click them:
![image](https://user-images.githubusercontent.com/19251112/123791588-ed91e400-d8df-11eb-9dbe-37bf3e30aa89.png)

Repo is available here: https://github.com/Kodular/Documentation, and live version at https://docs.kodular.io
Any idea of what can be causing this issue?

**Comment by squidfunk** [comment, marked as answer]
You seem to be having customizations in place (Carbon Ads) which might be causing this. If this behavior is reproducible without customizations, you can open an issue with a reproducible case, so we can fix it. However, the official documentation doesn't exhibit this behavior, so it's likely related to your customizations.

EDIT: Also make sure that you've followed [the upgrade guide](https://squidfunk.github.io/mkdocs-material/upgrading/#upgrading-from-6x-to-7x) and applied all changes when switching to a new major version.

**Reply by barreeeiroo** [reply]
True, thank you! We have actually overriden the `site_nav` block and the footer, and they were causing the issue. We upgraded from 5.x to 7.x without taking it into account. Thank you!
